1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,44 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+//variables
+const quoteP = document.querySelector("#quote");
+const  authorP = document.querySelector("#author");
+const  newQuoteBtn = document.querySelector("#new-quote");
+const  autoGenerate = document.querySelector(".checkbox");
+let intervalID;
+
+
+// eventListeners
+// Get a quote on load
+document.addEventListener("DOMContentLoaded", showQuoteOnLoad);
+// Get a quote on click
+newQuoteBtn.addEventListener("click", showQuoteOnClick);
+// Show quote automatically
+autoGenerate.addEventListener("click", showAuto);
+
+
+//functions
+// show quote on load
+function showQuoteOnLoad() {
+  const quoteOnLoad = pickFromArray(quotes);
+  quoteP.textContent = quoteOnLoad.quote;
+  authorP.textContent = `- ${quoteOnLoad.author}`;
+}
+// show quote on click
+function showQuoteOnClick() {
+  const quoteOnLoad = pickFromArray(quotes);
+  quoteP.textContent = quoteOnLoad.quote;
+  authorP.textContent = `- ${quoteOnLoad.author}`;
+}
```

Comment:
`showQuoteOnLoad()` and `showQuoteOnClick()` seem to be doing the same thing is there a way we could avoid this?